creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -254,33 +247,41 @@ async function _handleIssueSyncRequest({
         ensurePrimary: false
       })
 
-    const syncCorrectnessError = _ensureSyncsEnqueuedToCorrectNodes(
+    const syncCorrectnessAbort = _ensureSyncsEnqueuedToCorrectNodes(
       userReplicaSet,
       syncMode,
       syncRequestParameters.baseURL
     )
-    if (syncCorrectnessError) {
+    if (syncCorrectnessAbort) {
       return {
-        result: 'failure_sync_correctness',
-        error: {
-          message: `${logMsgString}: ${syncCorrectnessError}`
-        },
+        result: 'abort_sync_correctness',
+        abort: `${logMsgString}: ${syncCorrectnessAbort}`,
         syncReqsToEnqueue
       }
     }
 
-    const error = await primarySyncFromSecondary({
+    const response = await primarySyncFromSecondary({
       wallet: userWallet,
       secondary: secondaryEndpoint
     })
 
-    if (error) {
-      return {
-        result: 'failure_primary_sync_from_secondary',
-        error: {
```

Comment:
any reason you removed inner `message` field from error objects here? i kept it there so that it would mirror the `Error` class pattern and have `error.message` be defined

Reply:
k per slack conversation, confirmed this change has no impact since error is not consumed anywhere except for bull dashboard